src/Draco.Compiler/Syntax/Lexer.cs

```diff
@@ -374,6 +374,27 @@ private IToken LexString()
                 }
             }
             offset += mode.ExtendedDelims + 1;
+            // Line continuation
+            var whiteCharOffset = 0;
+            ch = this.Peek(offset);
+            if (char.IsWhiteSpace(ch))
+            {
+                while (char.IsWhiteSpace((ch = this.Peek(offset + whiteCharOffset))) && (ch != '\n')) whiteCharOffset++;
```

Comment:
Just use `IsSpace(this.Peek(offset + whiteCharOffset))` (we have these utility functions at the bottom)